src/shared/scrapers/BRA/county-level.js

```diff
@@ -0,0 +1,5621 @@
+import * as fetch from '../../lib/fetch/index.js';
+import * as parse from '../../lib/parse.js';
+import datetime from '../../lib/datetime/index.js';
+import * as transform from '../../lib/transform.js';
+import * as geography from '../../lib/geography/index.js';
+
+// Set county to this if you only have state data, but this isn't the entire state
+// const UNASSIGNED = '(unassigned)';
+
+const scraper = {
+  country: 'BRA',
+  type: 'csv',
+  priority: 1,
+  url: 'https://brasil.io/dataset/covid19/caso?place_type=city&format=csv',
+  timeseries: true,
+  aggregate: 'county',
+
+  _counties: [
```

Comment:
To keep this scraper small (and help with review!) would it be possible to move this county list to a JSON file and import it to here? You can just add this file to the directory where the scraper is.

Reply:
Sure thing! Great idea!